acid/acid3/numbered-tests.html

```diff
@@ -0,0 +1,26 @@
+<!doctype html>
+<title>Acid3 numbered tests</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script>
+var tests = undefined;
+
+function gotMessage(e) {
+  var m = e.data;
+  if (tests === undefined && "tests" in m) {
+    tests = [];
+    for (var i = 0; i < m.tests; i++) {
+      tests.push(async_test("Test " + i));
+    }
+  } else if ("result" in m) {
+    var test = m.test;
+    var passed = m.result === "pass";
+    var message = m.message;
+    tests[test].step_func_done(function() {
```

Comment:
I think the pattern of immediately calling the result of a `_func_` method is not very clear. Just do `step` + `done`?

Reply:
I agree.